docs/web/docs/1-getting_started/2-install_and_configure.mdx

```diff
@@ -77,10 +102,125 @@ iambic setup
+In a production deployment, IAMbic should be running in a CI/CD pipeline, triggered by a Github webhook.
+To facilitate this, IAMbic provides a Github Application that can be installed on your Github Organization or
+Github User account, a Docker container that will be deployed in AWS Lambda, and AWS EventBridge rules that
+will trigger the Lambda function to run IAMbic commands periodically.
+
+### Setup Github Application
+
+IAMbic uses a Github Application to manage the Github workflows that keep your IAM resources up to date.
+The Github Application is installed on your Github Organization or Github User account.
+
+* You need GitHub org admin permissions to create a GitHub app in your organization
+* You need AWS administrator permissions with the ability to assume IambicHubRole in the management account, and AWS administrator permissions to deploy the IAMbic GitHub integration.
+
+```bash
+iambic setup
+```
+
+* Select `Setup GitHub App Integration using AWS Lambda` option
+* When you install the application, give it access to your `iambic-templates`
+and `iambic-templates-gist` repositories. **Do not** give it access to your other repositories.
+
+See below for a video walkthrough of this flow:
+
+[![Setup IAMbic GitHub Integration](https://img.youtube.com/vi/3PikJZ0pW8A/0.jpg)](https://youtu.be/3PikJZ0pW8A)
+
+### Set the avatar of your GitHub App (Recommended)
+
+<details>
+  <summary>Expand for instructions</summary>
+
+Because your GitHub app will be making comments in your pull requests, you want
+to set the avatar of your GitHub app to easily recognize the identity.
+
+We have provide both light and dark icons for you.
+
+<img src={"/img/git/app-icons/icon-light.png"} style={{width: '25%', display: 'auto', marginLeft: 'auto', marginRight: 'auto'}} />
+<img src={"/img/git/app-icons/icon-dark.png"} style={{width: '25%', display: 'auto', marginLeft: 'auto', marginRight: 'auto'}} />
+
+Download them to your local computer.
+
+1. From the previous section, click on "App Settings".
+<img src={"/img/git/app-icons/step-1-app-settings.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+1. Scroll down to "Display Information", click on "Upload a logo..."
+<img src={"/img/git/app-icons/step-2-display.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+1. Choose either the light icon or dark icon from your computer
+1. Click on the "Set new avatar" button
+<img src={"/img/git/app-icons/step-3-set-avatar.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+
+</details>
+
+### Open a pull request on your **iambic-templates** repo
+
+1. `default_branch=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')`
+2. `git checkout -b task/change_description "$default_branch"`
+3. Edit one of your description within your templates, like `resources/github/iam_role_lambda.yaml`. <!-- #TODO edit to what?? -->
```

Comment:
I think we have an example for AWS quick start guide. Let's add this as example we re-use in the template repo. https://docs.iambic.org/getting_started/aws#3-create-and-manage-an-iam-role-across-multiple-aws-accounts